src/Microsoft.DotNet.Build.Tasks.Feed/PushToBlobFeed.cs

```diff
@@ -34,6 +41,22 @@ public class PushToBlobFeed : MSBuild.Task
 
         public int UploadTimeoutInMinutes { get; set; } = 5;
 
+        public bool SkipCreateManifest { get; set; }
+
+        public string ManifestName { get; set; } = "anonymous";
```

Comment:
Why not make this an error if not set instead of defaulting to "anonyomous"? That seems like it will be a costlier fix to try and understand that you needed to pass a name.

Reply:
OK that seems reasonable to me given the extra context.